src/main.cpp

```diff
@@ -211,15 +214,26 @@ void loop()
   float throttleRef = raspberryEsp32Interface.throttleReferencePWM();
 
   // Compute control actions and send them to the motors
-
+  #if UAV_TYPE == UAV_TYPE_QUADCOPTER
+  controller.motorPwmSignals(attitudeError,
+                             angularVelocityCorrected,
+                             yawRateReference,
+                             throttleRef,
+                             motorFL, motorFR, motorBL, motorBR,
+                             motorML, motorMR);
```

Comment:
`motorML` and `motorMR` are not (or, at least, should not be) defined for quadcopters